pennylane/ops/op_math/decompositions/controlled_decompositions.py

```diff
@@ -523,24 +553,33 @@ def _mcx_two_workers(wires, work_wires, work_wire_type, **__):
         `arXiv:2407.17966 <https://arxiv.org/abs/2407.17966>`__
 
     """
+    # pylint: disable=import-outside-toplevel
+    from pennylane.templates.subroutines.temporary_and import TemporaryAND
```

Comment:
Can we remove these imports and simply refer to `TemporaryAND` as `qml.TemporaryAND`?

Reply:
Nope, `tach` will not allow that.

Reply:
Oh wait. we did not clean `ops` up yet, so it won't! :o